pkg/types/ruletypes/api_params.go

```diff
@@ -98,11 +107,9 @@ func (r *PostableRule) processRuleDefaults() error {
 				}},
 			}
 			r.RuleCondition.Thresholds = &thresholdData
+			r.Evaluation = &EvaluationEnvelope{RollingEvaluation, RollingWindow{EvalWindow: r.EvalWindow, Frequency: r.Frequency}}
 		}
 	}
-	if r.Evaluation == nil {
-		r.Evaluation = &EvaluationEnvelope{RollingEvaluation, RollingWindow{EvalWindow: r.EvalWindow, Frequency: r.Frequency}}
-	}
```

Comment:
The `UnmarshalJSON` below calls the `processRuleDefaults` function and populates the thresholds and evaluation. There is no custom `MarshalJSON`; as a result, these changes get marshalled and sent in the response. Let's not do that. Add a custom marshal to skip the new fields marshal we added and plan to add if the alert is not new. The end is result is that there is no field or change to the payload for the existing alerts.